packages/ui/src/components/CourseProgressCard/CourseProgressCard.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+
+import { Box } from '../Box';
+import { CourseProgressCard, CourseProgressCardProps } from './CourseProgressCard';
+
+const status = [
```

Comment:
🙃 You can name it `statsData` and re-export to tests so you wouldn't duplicate mocks